routers/api.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
```

Comment:
@IgorChaikin Could we change project structure a little, we can create `server` directory, to store server initialization in `index.js`, also `routes` and `models`, and we can move `db.init.js` to `servers/script`.

Then `src` could be renamed to `client` to separate client and server code